kernel-modules/support-packages/05-create-index.py

```diff
@@ -73,11 +91,28 @@ def load_support_packages(output_dir, mod_md_map):
         support_pkg_file_latest = 'support-pkg-%s-latest.zip' % (mod_ver[:6])
```

Comment:
It looks like this truncation could create a collision if we were to have versions `1.10.1` and `1.10.10`.

Reply:
Good catch! I completely missed this when we changed the versioning of drivers! Think I can easily fix it, but I will do so in a follow up PR so we don't need to re-review this one.